src/crimson/os/seastore/transaction_manager.h

```diff
@@ -297,6 +297,11 @@ class TransactionManager : public ExtentCallbackInterface {
     Transaction &t,
     std::vector<laddr_t> offsets);
 
+  refs_ret dec_ref(
+    Transaction &t,
+    laddr_t start,
+    extent_len_t length);
```

Comment:
I don't see any user of this new interface.

Reply:
Sorry, this should have been removed.